release-content/migration-guides/handle_weak_replaced_with_handle_uuid.md

```diff
@@ -0,0 +1,26 @@
+---
+title: `Handle::Weak` has been replaced by `Handle::Uuid`.
+pull_requests: [19896]
+---
+
+`Handle::Weak` has some weird behavior. It allows for a sprite to be given a handle that is dropped
+**while the sprite is still using it**. This also results in more complexity in the asset system.
+The primary remaining use for `Handle::Weak` is to store asset UUIDs initialized through the
+`weak_handle!` macro. To address this, `Handle::Weak` has been replaced by `Handle::Uuid`!
+
+Users using the `weak_handle!` macro should switch to the `uuid_handle!` macro.
+
+```rust
+# Before
+const IMAGE: Handle<Image> = weak_handle!("12345678-9abc-def0-1234-56789abcdef0");
```

Comment:
```suggestion
const IMAGE: Handle<Image> = weak_handle!("b20988e9-b1b9-4176-b5f3-a6fa73aa617f");
```
Let's use a real UUID instead of a synthetic one here, otherwise a reader might get the impression they are meant to smash their keyboard manually ;)

Reply:
Oh you're supposed to generate them? I would be surprised if users actually did that hahaha.

I don't care enough, so done.